lib/puppet/util/run_mode.rb

```diff
@@ -38,6 +38,10 @@ def log_dir
         "$vardir/log"
       end
 
+      def conf_dir
+        "$puppetdir/config"
+      end
+
```

Comment:
Yeah, I think I agree with that statement, but it seems like it would imply that the log_dir value a few lines earlier should be equally broken.